src/dmd/mars.d

```diff
@@ -1554,7 +1572,7 @@ private bool parseCommandLine(const ref Strings arguments, const size_t argc, re
                 params.useDeprecated = 2;
             else if (strcmp(p + 1, "c") == 0)  // https://dlang.org/dmd.html#switch-c
                 params.link = false;
-            else if (memcmp(p + 1, cast(char*)"color", 5) == 0) // https://dlang.org/dmd.html#switch-color
+            else if (startsWith(p + 1, "color") == 0) // https://dlang.org/dmd.html#switch-color
```

Comment:
If you're returning a `bool`, then you should be able to use `if (startsWith(p + 1, "color"))` without the comparison to `0`.

Reply:
yes, I know :-)